### PR TITLE
Update instanceID

### PR DESCRIPTION
**Pull request by dwertent**
## Overview
We are using instancesID (podSpecID) as defined [here](https://github.com/kubescape/k8s-interface/blob/update-instanceid/instanceidhandler/interface.go)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,7 +17,7 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.5.0
 	github.com/kubescape/go-logger v0.0.11
-	github.com/kubescape/k8s-interface v0.0.112
+	github.com/kubescape/k8s-interface v0.0.114-0.20230326195717-d11a8f35efc2
```
Comment (matthyx) [inline go.mod:20]: you don't want to wait for the released version?

File: mainhandler/vulnscan.go [hunk 2]
```diff
@@ -287,6 +287,15 @@ func (actionHandler *ActionHandler) scanWorkload(ctx context.Context, sessionObj
 		logger.L().Info(fmt.Sprintf("workload %s has no podSpec, skipping", actionHandler.wlid))
 		return nil
 	}
+
+	// get pod instanceID
+	instancesID, err := instanceidhandler.GenerateInstanceIDFromPod(pod)
```
Comment (matthyx) [inline mainhandler/vulnscan.go:292]: instanceIDs ? I'm not sure about plurals for composed words.

**Review by matthyx — COMMENTED** [review]
maybe let's wait for the released lib

File: mainhandler/vulnscan.go (hunk 2 above)
Comment (dwertent) [reply mainhandler/vulnscan.go]: Resolved in [this](https://github.com/kubescape/operator/pull/99/commits/7f88b33efd8b0c5d9c0d7d00961022b607d96311) commit

**Review by matthyx — APPROVED** [review]

**Pull request merged**